pages/dashboard.js

```diff
@@ -152,7 +152,6 @@ const Dashboard = () => {
             Looks like you haven't installed the package manager wrapper yet.
             Head over to{' '}
             <TextLink
-              textDecoration='underline'
```

Comment:
i personally find it hard to tell it's a link without the underline since the text is the same color as the surrounding text. what do you think?

Reply:
pete has a change to this that helps more by making the outter text "rock" color